Tasks/UseRubyVersion/userubyversion.ts

```diff
@@ -0,0 +1,101 @@
+import * as fs from 'fs';
+import * as os from 'os';
+import * as path from 'path';
+
+import * as semver from 'semver';
+
+import * as task from 'vsts-task-lib/task';
+import * as tr from 'vsts-task-lib/toolrunner';
+import * as tool from 'vsts-task-tool-lib/tool';
+
+export enum Platform {
+    Windows,
+    MacOS,
+    Linux
+}
+
+export function getPlatform(): Platform {
+    switch (process.platform) {
+        case 'win32': return Platform.Windows;
+        case 'darwin': return Platform.MacOS;
+        case 'linux': return Platform.Linux;
+        default: throw Error(task.loc('PlatformNotRecognized'));
+    }
+}
```

Comment:
Seem would be useful in other places, consider if this should be pulled into a common helper or should be part of the tool-lib itself

Reply:
I've been meaning to move this to task-lib, we're wanting a nice way to determine the platform